### PR TITLE
Add extra storage to store search result exports.

### PR DESCRIPTION
**Pull request by ewelinajozwik**
Implements [FNS-36](https://thehyve.atlassian.net/browse/FNS-36), including:

- fix for multiselection from [FNS-37](https://thehyve.atlassian.net/browse/FNS-37)
- [FNS-38](https://thehyve.atlassian.net/browse/FNS-38) (Analysis button)
- [FNS-43](https://thehyve.atlassian.net/browse/FNS-43 )(Extra storage API)

Linked PR related to [FNS-44](https://thehyve.atlassian.net/browse/FNS-44) (Jupyter extra storage mounting): https://github.com/thehyve/fairspace-jupyter/pull/32 .

[FNS-36]: https://thehyve.atlassian.net/browse/FNS-36?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FNS-37]: https://thehyve.atlassian.net/browse/FNS-37?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FNS-38]: https://thehyve.atlassian.net/browse/FNS-38?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FNS-43]: https://thehyve.atlassian.net/browse/FNS-43?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FNS-44]: https://thehyve.atlassian.net/browse/FNS-44?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -3014,6 +3014,10 @@ directory listings and file properties handling, adding, moving, replacing and d
 There are two types of storages that Saturn connects to: primary for storing RDF triples and file blobs,
 and secondary for efficient data retrieval.
 
+There is also a possibility to configure an extra file storage for Saturn,
+separate from the primary one to keep i.e. search result exports created by users,
```
Comment (frankyhollywood) [inline README.adoc]:
```suggestion
separate from the primary one to keep e.g. search result exports created by users,
```

File: README.adoc [hunk 2]
```diff
@@ -3040,6 +3044,21 @@ To improve the performance of metadata views, especially for counts and paginati
 (PostgreSQL) is used. This database is flexible, created based on views configuration file and RDF storage,
 which enables recreation of the database at any point of time.
 
+==== Extra file storage
+
+Similar to the primary file storage, the extra storage is an virtual file system
```
Comment (frankyhollywood) [inline README.adoc]:
```suggestion
Similar to the primary file storage, the extra storage is a virtual file system
```

File: charts/fairspace/values.yaml [hunk 3]
```diff
@@ -72,6 +72,7 @@ saturn:
       size: 50Gi
     audit:
       size: 5Gi
+    extra-file-storage:
```
Comment (frankyhollywood) [inline charts/fairspace/values.yaml:75]: no true or false?

**Review by frankyhollywood — APPROVED** [review]

File: charts/fairspace/values.yaml (hunk 3 above)
Comment (ewelinajozwik) [reply charts/fairspace/values.yaml:75]: no, this is just a template for putting other extra-file-storage configuration, like e.g. size. It is basically ` extra-file-storage={}` and it doesn't end up in the final config that is built from this template, when it's empty

**Comment by ewelinajozwik** [comment]
@FrankyHollywood could you also have a look at the linked PR: [FNS-44](https://thehyve.atlassian.net/browse/FNS-44)

Also, I just realized that I forgot to add some loading indicator after clicking the "export to analysis" button, so I'll merge this PR once I add it...

[FNS-44]: https://thehyve.atlassian.net/browse/FNS-44?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Pull request merged**